### PR TITLE
potential fix to workspace provider upgrade command

### PR DESCRIPTION
**Pull request by bpmct**
Context: I remember we removed `--force` from our standard update as it was constantly failing the update as described here: https://coder.com/docs/setup/updating#fixing-a-failed-upgrade: https://github.com/cdr/docs/pull/171

When updating my workspace provider, I ran into similar errors and my deployment succeeded without the command. I'm not exactly sure why, but I believe remember seeing `--force` only leads to a successful install on specific versions of helm. A sanity check would help, though.

### CODE REVIEW

**Review by vapurrmaid — COMMENTED** [review]
Changes look good to me but @f0ssel should give the :heavy_check_mark:

**Comment by f0ssel** [comment]
@sreya is this the general rec going forward? I can remove it in the coder-cli output as well

**Comment by khorne3** [comment]
@sreya Do you mind taking a look at the questions in this PR, please? Thanks!

**Comment by bpmct** [comment]
bump @sreya

File: workspaces/variables.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
 ---
-title: Workspace variables
-description: Learn how to work with CODER_* workspace variables
+title: Environment variables
```
Comment (jawnsy) [inline workspaces/variables.md:2]: I think these changes should be in a separate PR to keep the history clean

File: admin/workspace-providers/deployment.md [hunk 2]
```diff
@@ -158,8 +158,7 @@ values changes; you can do this with
 helm upgrade coder-workspace-provider coder/workspace-provider \
     --version=[CODER_VERSION] \
     --atomic \
-    --install \
-    --force
+    --install
```
Comment (jawnsy) [inline admin/workspace-providers/deployment.md:161]: We no longer use `--force` for our internal deploys (preview, kind, dogfood, workspace providers) so I think it makes sense to update the docs accordingly, yeah.

**Comment by sreya** [comment]
I guess this got lost in my barrage of emails but yes we no longer suggest using `--force`

File: workspaces/variables.md (hunk 1 above)
Comment (bpmct) [reply workspaces/variables.md:2]: unsure how this got into here in the first place. thanks

**Review by deansheather — APPROVED** [review]

**Comment by deansheather** [comment]
uh oh I didn't realize this was auto-merge 😬

**Comment by bpmct** [comment]
all good @deansheather. i think we wanted to merge it.
